sycl/test/tracing/buffer_printers.cpp

```diff
@@ -0,0 +1,55 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
+// RUN: env SYCL_PI_TRACE=2 %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
+// RUN: env SYCL_PI_TRACE=2 %CPU_RUN_PLACEHOLDER %t.out %CPU_CHECK_PLACEHOLDER
```

Comment:
According to https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#test-dpc-toolchain the tests should go to  LLVM* test suite.

Reply:
Removed tests. Prepared PR https://github.com/intel/llvm-test-suite/pull/64 to add tests to llvm-test-suite